ibis/backends/pandas/client.py

```diff
@@ -135,6 +136,13 @@ def convert_boolean_to_series(in_dtype, out_dtype, column):
     return column
 
 
+@sch.convert.register(DatetimeTZDtype, dt.Date, pd.Series)
```

Comment:
Since we support more output types now I think we should factor out `sch.convert()` from `ibis.expr.schema` directly to the pandas backend.

Reply:
Should we do that in a follow-up? I'm not 100% sure how we should do it though, but happy to make an issue if that's helpful!

Reply:
Sure, follow-up sgtm.